test/partition_key_tests.py

```diff
@@ -0,0 +1,187 @@
+# The MIT License (MIT)
+# Copyright (c) 2014 Microsoft Corporation
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import unittest
+import pytest
+import requests
+import datetime
+import six
+import json
+import uuid
+from six.moves.urllib.parse import quote as urllib_quote
+import azure.cosmos.auth as auth
+import azure.cosmos.partition_key as partition_key
+import azure.cosmos.cosmos_client as cosmos_client
+import test.test_config as test_config
+
+
+@pytest.mark.usefixtures("teardown")
+class PartitionKeyTests(unittest.TestCase):
+    """Tests to verify if non partitoned collections are properly accessed on migration with version 2018-12-31.
+    """
+
+    host = test_config._test_config.host
+    masterKey = test_config._test_config.masterKey
+    connectionPolicy = test_config._test_config.connectionPolicy
+    client = cosmos_client.CosmosClient(host, {'masterKey': masterKey}, "Session", connectionPolicy)
+    created_db = test_config._test_config.create_database_if_not_exist(client)
+    created_collection = test_config._test_config.create_multi_partition_collection_with_custom_pk_if_not_exist(client)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.created_db.delete_container(container=cls.created_collection_id)
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
```suggestion
    def set_up_class(cls):
```

Reply:
overrides baseclass method from unittest.Testcase, should not be renamed